### ISSUE TITLE
Reflections On appears to give all objects/terrain a white sheen

### ISSUE THREAD

**Issue by DemonDog47** [StrangeLoopGames/EcoIssues#18729]
Comparison shot at 3 different times of day provided of reflections on and off. Effect most prominent at night, particularly on modded objects. Also shown is that certain metallic surfaces turn completely black at certain times if reflections are off. (Try it with road barrier forms for a particularly egregious example) 

![20201013014321_1](https://user-images.githubusercontent.com/72786287/95827051-dc288d80-0cf8-11eb-8bb4-808fff99b8b4.jpg)
![20201013014331_1](https://user-images.githubusercontent.com/72786287/95827054-dcc12400-0cf8-11eb-9955-4bf8c4bcb0a3.jpg)
![20201013014359_1](https://user-images.githubusercontent.com/72786287/95827056-dcc12400-0cf8-11eb-958d-6301d741fb1a.jpg)
![20201013014415_1](https://user-images.githubusercontent.com/72786287/95827059-dd59ba80-0cf8-11eb-8d45-000eb3584369.jpg)
![20201013014428_1](https://user-images.githubusercontent.com/72786287/95827061-ddf25100-0cf8-11eb-897a-c4f9290182cf.jpg)
![20201013014434_1](https://user-images.githubusercontent.com/72786287/95827064-ddf25100-0cf8-11eb-8932-bab0ced5c4f6.jpg)
![20201013021400_1](https://user-images.githubusercontent.com/72786287/95827754-c7002e80-0cf9-11eb-86f4-b4a1ece144b5.jpg)

**Comment by IcedForge** [comment]
Should be mentioned that any of these modded objects are not supported officially so they would need to update/fix this on their end. But the overall aspect needs to be reviewed.

**Comment by DemonDog47** [comment]
Yeah, I acknowledge modded stuff is sort of out of the purview of development, but it was the first place I noticed it at so I figured worth mentioning. I've seen the effect on it on several mods from different creators, so I'm not sure how to really fix it on the mod end. Here's another comparison screenshot at midnight highlighting the problem on terrain and the effect on metallic objects.
![20201014182915_1](https://user-images.githubusercontent.com/72786287/96056035-35083b00-0e4b-11eb-9161-afc3b1e4b600.jpg)
![20201014182919_1](https://user-images.githubusercontent.com/72786287/96056037-35a0d180-0e4b-11eb-9c59-91822e4c484c.jpg)

**Comment by keegan-o** [comment]
This can be affected by problems with the reflection probe ending up inside voxels, for instance in any building interior seeing the difference between when you are right next to/up against a wall, versus in the middle of the room, you can see the reflection probe updating dramatically which is bad.
